.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Can you remote these files under .terraform? I think they're the source of the issue.

Reply:
Done